repositories.yaml

```diff
@@ -50,6 +50,12 @@ repositories:
     url: stackabletech/druid-operator.git
     product_string: druid
     pretty_string: Apache Druid
+  - name: secret-operator
+    url: stackabletech/secret-operator.git
+    product_string: secret-operator
+    pretty_string: Stackable Secret Operator
+    run_as: custom
+    include_productconfig: false
 
 # Anything specified in the retired_files variable will be deleted.
 # This is uncommented as I had issues with everything being deleted when this was just present as an empty key.
```

Comment:
We need to specify the 'deploy/crd' folder here I think, otherwise the files in there will remain in the repos as orphans.

Reply:
Dupe of https://github.com/stackabletech/operator-templating/pull/55#discussion_r827876855.

Reply:
oh man, sorry for being so absent minded